### PR TITLE
#AB 50 Allows removal of Users

### PR DESCRIPTION
**Pull request by joenoon16510**

### CODE REVIEW

File: src/pages/EditOrganization.tsx [hunk 1]
```diff
@@ -121,13 +121,20 @@ const UsersTable = () => {
     .catch(() => history.push('/dashboard'));
   }, []);
 
+  const onClickRemoveUser = (removedUser:OrganizationUser) =>
+  {
+    OrganizationService.deleteOrganizationUser(removedUser.organizationId,removedUser.userId);
+    window.location.reload();
```
Comment (zlix12) [inline src/pages/EditOrganization.tsx:127]: It may be better to just remove the deleted user from the list than reload the whole page.

**Review by zlix12 — CHANGES_REQUESTED** [review]

File: src/pages/EditOrganization.tsx [hunk 2]
```diff
@@ -121,13 +123,19 @@ const UsersTable = () => {
     .catch(() => history.push('/dashboard'));
   }, []);
 
+  const onClickRemoveUser = (removedUser:OrganizationUser) =>
+  {
+    OrganizationService.deleteOrganizationUser(removedUser.organizationId,removedUser.userId);
```
Comment (zlix12) [inline src/pages/EditOrganization.tsx:128]: You still need to remove the user from the displayed list. For example:

```suggestion
    OrganizationService.deleteOrganizationUser(removedUser.organizationId, removedUser.userId).then(() => setOrgUsers(orgUsers.filter((orgUser) => orgUser.id !== removedUser.userId)));
```

**Review by zlix12 — CHANGES_REQUESTED** [review]

File: src/pages/EditOrganization.tsx [hunk 3]
```diff
@@ -142,6 +150,19 @@ const UsersTable = () => {
               }
             </Td>
             <Td isNumeric>{orgUser.userId}</Td>
+            <Td>
+              <Flex justifyContent = "flex-end">
+                <Button colorScheme = "green">
```
Comment (zlix12) [inline src/pages/EditOrganization.tsx:155]: If this confirm button is not doing anything we should probably remove it from this PR.

**Review by zlix12 — CHANGES_REQUESTED** [review]

File: src/pages/EditOrganization.tsx (hunk 3 above)
Comment (kpjVideo) [reply src/pages/EditOrganization.tsx:155]: Agreed. A confirmation popover would be sufficient, then I think we can approve this branch for master.

**Review by kpjVideo — APPROVED** [review]
Functionally, looks good. Though maybe worth getting a substitute for the confirmation before merging.

**Review by zlix12 — APPROVED** [review]

**Pull request merged**